src/BizHawk.WinForms.Controls/BizHawk.WinForms.Controls.csproj

```diff
@@ -10,4 +10,9 @@
 		<PackageReference Include="System.Drawing.Common" />
 		<ProjectReference Include="$(ProjectDir)../BizHawk.Common/BizHawk.Common.csproj" />
 	</ItemGroup>
+	<ItemGroup>
+	  <Compile Update="LabelEx\HexLabelEx.cs">
+	    <SubType>Component</SubType>
+	  </Compile>

```

Comment:
Does this have any effect? If so then I'd expect the rest of the controls in this project to be missing their IDE icons or whatever, since they're obviously not listed here.